wdl/src/bin/wdl.rs

```diff
@@ -103,7 +105,15 @@ async fn analyze<T: AsRef<dyn Rule>>(
         },
     );
 
-    analyzer.add_documents(vec![path]).await?;
+    let file = if let Ok(url) = Url::parse(&file) {
+        url
+    } else if let Some(url) = path_to_uri(&PathBuf::from(file.clone())) {
```

Comment:
```suggestion
    } else if let Some(url) = path_to_uri(&Path::new(&file)) {
```

Or simply `path_to_uri(&file)` if we change `path_to_uri` to take `impl AsRef<Path>`.